codespell_lib/data/dictionary.txt

```diff
@@ -6377,22 +6377,39 @@ azma->asthma
 azmith->azimuth
 azmiths->azimuths
 baase->base, abase,
```

Comment:
Perhaps:
```
baase->base, abase, baaes,
```

And add other verbals forms of `baa`elsewhere.

Reply:
`baa` and similar are too uncommon.